tests/decode-unknown-2/test.yaml

```diff
@@ -0,0 +1,22 @@
+requires:
+    min-version: 8
+
+args:
+- -k none
+
+pcap: ../decode-unknown-1/input.pcap
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: stats
+  - stats:
+      decoder.ethernet: 1
+      decoder.unknown_ethertype: 1
+  - filter:
+      count: 1
+      match:
+        event_type: anomaly
+        anomaly.type: decode
+        anomaly.event: decoder.ethernet.unknown_ethertype
```

Comment:
do we have the unknown ether type logged ?

Reply:
No, the unknown value is not logged

Reply:
Hmmm... How could we get it ?

Reply:
I guess pcap logging would be one way to get it...

Reply:
Is not `ether.ether_type: 47099` what we want ?